woocommerce/woocommerce-android
# issue/1636-blank-product-list

Fixes #1636 - To test:

* Enable "Don't keep activities"
* Go to the product list
* Open settings
* Return to the product list
* Ensure the product list is populated (previously it would be empty)

Update release notes:
